hiero-dependency-versions/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ val jackson = "2.18.3"
 val junit5 = "5.10.3!!" // no updates beyond 5.10.3 until #17125 is resolved
 val log4j = "2.24.3"
 val mockito = "5.17.0"
-val pbj = "0.11.0" // ATTENTION: keep in sync with plugin version in 'hapi/hapi/build.gradle.kts'
+val pbj = "0.11.3" // ATTENTION: keep in sync with plugin version in 'hapi/hapi/build.gradle.kts'
```

Comment:
Please also manually upgrade the version in `hapi/hapi/build.gradle.kts` as it says in the comment.